### PR TITLE
Add new tile features

### PR DESCRIPTION
**Pull request by piitaya**
## Proposed change

Add new tile features : 
- climate HVAC modes 
- cover position
- cover tilt position
- lawn mower commands
- light color temp
- target temperature 
- water heater operation modes

I also updated tile card screenshot to better highlight tile features.

CleanShot 2023-08-29 at 10 53 22

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/frontend/pull/17697, https://github.com/home-assistant/frontend/pull/17655, https://github.com/home-assistant/frontend/pull/17619, https://github.com/home-assistant/frontend/pull/17597, https://github.com/home-assistant/frontend/pull/17592, https://github.com/home-assistant/frontend/pull/16515, https://github.com/home-assistant/frontend/pull/16110
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_dashboards/tile.markdown [hunk 1]
```diff
@@ -138,6 +138,39 @@ modes:
   type: list
 {% endconfiguration %}
 
+### Climate HVAC modes
+
+Widget that display buttons to control change the HVAC mode for a [climate](/integrations/climate).
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
Widget that displays buttons to control the HVAC mode for a [climate](/integrations/climate).
```

File: source/_dashboards/tile.markdown [hunk 2]
```diff
@@ -138,6 +138,39 @@ modes:
...
+
+<p class='img'>
+  <img src='/images/dashboards/tile-features/climate_hvac_modes.png' alt='Screenshot of the tile card with climate HVAC modes feature'>
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
  <img src='/images/dashboards/tile-features/climate_hvac_modes.png' alt='Screenshot of the tile card with the climate HVAC modes feature'>
```

File: source/_dashboards/tile.markdown [hunk 3]
```diff
@@ -138,6 +138,39 @@ modes:
...
+  Screenshot of the tile card with climate HVAC modes feature
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
  Screenshot of the tile card with the climate HVAC modes feature
```

File: source/_dashboards/tile.markdown [hunk 4]
```diff
@@ -138,6 +138,39 @@ modes:
...
+  type: string
+hvac_modes:
+  required: true
+  description: List of modes to show on the card. The list can contain `auto`, `heat_cool`, `heat`, `cool`, `dry`, `fan_only` and `off`.
+  type: list
+{% endconfiguration %}
+
 ### Cover open/close
 
 Widget that display buttons to open, close or stop a [cover](/integrations/cover).
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
Widget that displays buttons to open, close, or stop a [cover](/integrations/cover).
```

File: source/_dashboards/tile.markdown [hunk 5]
```diff
@@ -159,6 +192,27 @@ type:
   type: string
 {% endconfiguration %}
 
+### Cover position
+
+Widget that display a slider to control the position for a [cover](/integrations/cover).
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
Widget that displays a slider to control the position for a [cover](/integrations/cover).
```

File: source/_dashboards/tile.markdown [hunk 6]
```diff
@@ -159,6 +192,27 @@ type:
...
+
+<p class='img'>
+  <img src='/images/dashboards/tile-features/cover_position.png' alt='Screenshot of the tile card with cover position feature'>
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
  <img src='/images/dashboards/tile-features/cover_position.png' alt='Screenshot of the tile card with the cover position feature'>
```

File: source/_dashboards/tile.markdown [hunk 7]
```diff
@@ -159,6 +192,27 @@ type:
...
+  Screenshot of the tile card with cover position feature
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
  Screenshot of the tile card with the cover position feature
```

File: source/_dashboards/tile.markdown [hunk 8]
```diff
@@ -159,6 +192,27 @@ type:
...
+{% configuration %}
+type:
+  required: true
+  description: "`cover-position`"
+  type: string
+{% endconfiguration %}
+
 ### Cover tilt
 
 Widget that display buttons to open, close or stop a [cover](/integrations/cover).
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
Widget that displays buttons to open, close, or stop a [cover](/integrations/cover).
```

File: source/_dashboards/tile.markdown [hunk 9]
```diff
@@ -180,6 +234,27 @@ type:
   type: string
 {% endconfiguration %}
 
+### Cover tilt position
+
+Widget that display a slider to control the tilt position for a [cover](/integrations/cover).
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
Widget that displays a slider to control the tilt position for a [cover](/integrations/cover).
```

File: source/_dashboards/tile.markdown [hunk 10]
```diff
@@ -180,6 +234,27 @@ type:
...
+
+<p class='img'>
+  <img src='/images/dashboards/tile-features/cover_tilt_position.png' alt='Screenshot of the tile card with cover tilt position feature'>
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
  <img src='/images/dashboards/tile-features/cover_tilt_position.png' alt='Screenshot of the tile card with the cover tilt position feature'>
```

File: source/_dashboards/tile.markdown [hunk 11]
```diff
@@ -180,6 +234,27 @@ type:
...
+  Screenshot of the tile card with cover tilt position feature
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
  Screenshot of the tile card with the cover tilt position feature
```

File: source/_dashboards/tile.markdown [hunk 12]
```diff
@@ -180,6 +234,27 @@ type:
...
+{% configuration %}
+type:
+  required: true
+  description: "`cover-tilt-position`"
+  type: string
+{% endconfiguration %}
+
 ### Fan speed
 
 Widget that display speed controls for a [fan](/integrations/fan).
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
Widget that displays speed controls for a [fan](/integrations/fan).
```

File: source/_dashboards/tile.markdown [hunk 13]
```diff
@@ -201,6 +276,34 @@ type:
   type: string
 {% endconfiguration %}
 
+### Lawn mower commands
+
+Widget that display buttons to control a [lawn mower](/integrations/lawn_mower).
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
Widget that displays buttons to control a [lawn mower](/integrations/lawn_mower).
```

File: source/_dashboards/tile.markdown [hunk 14]
```diff
@@ -201,6 +276,34 @@ type:
...
+
+<p class='img'>
+  <img src='/images/dashboards/tile-features/lawn_mower_commands.png' alt='Screenshot of the tile card with lawn mower commands feature'>
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
  <img src='/images/dashboards/tile-features/lawn_mower_commands.png' alt='Screenshot of the tile card with the lawn mower commands feature'>
```

File: source/_dashboards/tile.markdown [hunk 15]
```diff
@@ -201,6 +276,34 @@ type:
...
+  Screenshot of the tile card with lawn mower commands feature
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
  Screenshot of the tile card with the lawn mower commands feature
```

File: source/_dashboards/tile.markdown [hunk 16]
```diff
@@ -201,6 +276,34 @@ type:
...
+  type: string
+commands:
+  required: true
+  description: List of commands to show on the card. The list can contain `start_pause` and `dock`.
+  type: list
+{% endconfiguration %}
+
 ### Light brightness
 
 Widget that display a slider to select the brightness for a [light](/integrations/light).
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
Widget that displays a slider to select the brightness for a [light](/integrations/light).
```

File: source/_dashboards/tile.markdown [hunk 17]
```diff
@@ -222,6 +325,48 @@ type:
   type: string
 {% endconfiguration %}
 
+### Light color temp
+
+Widget that display a slider to select the color temperature for a [light](/integrations/light).
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
Widget that displays a slider to select the color temperature for a [light](/integrations/light).
```

File: source/_dashboards/tile.markdown [hunk 18]
```diff
@@ -222,6 +325,48 @@ type:
...
+
+<p class='img'>
+  <img src='/images/dashboards/tile-features/light_color_temp.png' alt='Screenshot of the tile card with light color temperature feature'>
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
  <img src='/images/dashboards/tile-features/light_color_temp.png' alt='Screenshot of the tile card with the light color temperature feature'>
```

File: source/_dashboards/tile.markdown [hunk 19]
```diff
@@ -222,6 +325,48 @@ type:
...
+  Screenshot of the tile card with light color temperature feature
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
  Screenshot of the tile card with the light color temperature feature
```

File: source/_dashboards/tile.markdown [hunk 20]
```diff
@@ -222,6 +325,48 @@ type:
...
+{% configuration %}
+type:
+  required: true
+  description: "`light-color-temp`"
+  type: string
+{% endconfiguration %}
+
+### Target temperature
+
+Widget that display buttons to select the target temperature for a [climate](/integrations/climate) or a [water heater](/integrations/water_heater).
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
Widget that displays buttons to select the target temperature for a [climate](/integrations/climate) or a [water heater](/integrations/water_heater).
```

File: source/_dashboards/tile.markdown [hunk 21]
```diff
@@ -222,6 +325,48 @@ type:
...
+
+<p class='img'>
+  <img src='/images/dashboards/tile-features/target_temperature.png' alt='Screenshot of the tile card with target temperature feature'>
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
  <img src='/images/dashboards/tile-features/target_temperature.png' alt='Screenshot of the tile card with the target temperature feature'>
```

File: source/_dashboards/tile.markdown [hunk 22]
```diff
@@ -222,6 +325,48 @@ type:
...
+  Screenshot of the tile card with target temperature feature
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
  Screenshot of the tile card with the target temperature feature
```

File: source/_dashboards/tile.markdown [hunk 23]
```diff
@@ -222,6 +325,48 @@ type:
...
+{% configuration %}
+type:
+  required: true
+  description: "`target-temperature`"
+  type: string
+{% endconfiguration %}
+
 ### Vacuum commands
 
 Widget that display buttons to control a [vacuum](/integrations/vacuum).
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
Widget that displays buttons to control a [vacuum](/integrations/vacuum).
```

File: source/_dashboards/tile.markdown [hunk 24]
```diff
@@ -252,3 +397,36 @@ commands:
   description: List of commands to show on the card. The list can contain `start_pause`, `stop`, `clean_spot`, `locate` and `return_home`.
   type: list
 {% endconfiguration %}
+
+### Water heater operation modes
+
+Widget that display buttons to control change the operation mode of a [water heater](/integrations/water_heater).
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
Widget that displays buttons to control the operation mode of a [water heater](/integrations/water_heater).
```

File: source/_dashboards/tile.markdown [hunk 25]
```diff
@@ -252,3 +397,36 @@ commands:
...
+
+<p class='img'>
+  <img src='/images/dashboards/tile-features/water_heater_operation_modes.png' alt='Screenshot of the tile card with water heater operation modes feature'>
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
  <img src='/images/dashboards/tile-features/water_heater_operation_modes.png' alt='Screenshot of the tile card with the water heater operation modes feature'>
```

File: source/_dashboards/tile.markdown [hunk 26]
```diff
@@ -252,3 +397,36 @@ commands:
...
+  Screenshot of the tile card with water heater operation modes feature
```
Comment (c0ffeeca7) [inline source/_dashboards/tile.markdown]:
```suggestion
  Screenshot of the tile card with the water heater operation modes feature
```

**Review by c0ffeeca7 — APPROVED** [review]
Looks nice :star_struck: . Thank you, @piitaya :+1:

**Comment by piitaya** [comment]
Amazing! Thanks a lot for the changes! 😀

**Pull request merged**